### PR TITLE
Migrate ACM prow pipelines to GHA

### PR DESCRIPTION
**Pull request by m00g3n**
**Description**
Migrate ACM prow pipelines to GHA.

Changes proposed in this pull request:
Adapt the following tests to run on GHA 
- application-conn-validator
- application-gateway
- compass-runtime-agent

Links:
- [ACM tests in kyma repo](https://github.com/kyma-project/kyma/tree/main/tests/components/application-connector)

**Related issue(s)**

- https://github.com/kyma-project/application-connector-manager/issues/104
- https://github.com/kyma-project/application-connector-manager/issues/17

### CODE REVIEW

File: tests/resources/charts/application-connectivity-validator-test/values.yaml [hunk 1]
```diff
@@ -0,0 +1,15 @@
+# Default values for application-connectivity-validator-test.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+global:
+  containerRegistry:
+    path: "europe-docker.pkg.dev/kyma-project"
```
Comment (koala7659) [inline tests/resources/charts/application-connectivity-validator-test/values.yaml:7]: Please update repo name to local k3d repository as we do for compass-runtime-agent tests

File: tests/resources/charts/gateway-test/values.yaml [hunk 2]
```diff
@@ -0,0 +1,19 @@
+global:
+  containerRegistry:
+    path: "europe-docker.pkg.dev/kyma-project"
```
Comment (koala7659) [inline tests/resources/charts/gateway-test/values.yaml:3]: Please update repo name and image to use local k3d repository as we do for compass-runtime-agent tests

**Review by koala7659 — CHANGES_REQUESTED** [review]

File: tests/resources/charts/application-connectivity-validator-test/values.yaml (hunk 1 above)
Comment (m00g3n) [reply tests/resources/charts/application-connectivity-validator-test/values.yaml:7]: as we agreed offline, this issue will be addressed in separate PR

File: tests/resources/charts/gateway-test/values.yaml (hunk 2 above)
Comment (m00g3n) [reply tests/resources/charts/gateway-test/values.yaml:3]: as we agreed offline, this issue will be addressed in separate PR

**Review by koala7659 — APPROVED** [review]
LGTM

**Review by grego952 — APPROVED** [review]

**Pull request merged**